OracleWebLogic/dockerfiles/12.2.1.4/Dockerfile.generic

```diff
@@ -91,17 +94,17 @@ ENV DOMAIN_NAME="${DOMAIN_NAME:-base_domain}" \
 # ------------------------------------------------------------
 RUN mkdir -p /u01 && \
     chmod a+xr /u01 && \
+    chmod -R g+rwX /u01 && \
```

Comment:
There is nothing in /u01, you just created it two lines prior.  There is no need for a -R.  Combine lines 96 and 97:
   chmod 775 /u01 &&